qoqo_qasm/tests/integration/interface.rs

```diff
@@ -108,7 +108,6 @@ fn test_qasm_call_operation(operation: Operation, converted: &str) {
 
 /// Test qasm_call_operation and qasm_call_circuit errors
 #[test_case(Operation::from(ISwap::new(0, 1)))]
-#[test_case(Operation::from(ControlledPhaseShift::new(0, 1, CalculatorFloat::from(0.23))))]
 #[test_case(Operation::from(FSwap::new(0, 1)))]
```

Comment:
How come this has been deleted?

Reply:
ControlledPhaseShift has been implemented, so it shouldn't trigger an error

Reply:
Ah fair, has it been added to the tests that should pass?

Reply:
Similarly, have the new gates you added been added too? I can't see them in the test